src/components/DashboardBody.js

```diff
@@ -49,20 +49,24 @@ const dashboardBodyProps = {
     },
   ]
 }
+const BuildWidgets = widgetArray => (
```

Comment:
Don't need the parens at the end of the line here (and corresponding ending parens)